docs/how-to/mgc-setup.md

```diff
@@ -0,0 +1,145 @@
+# Setting up MGC in Existing OCM Clusters
+
+This guide will show you how to install and configure the Multi-Cluster Gateway Controller in preexisting [Open Cluster Management](https://open-cluster-management.io/)-configured clusters.
+
+## Prerequisites
+
+- A **hub cluster** running the OCM control plane (v0.11.0 or greater)
+- Any number of additional **spoke clusters** that have been configured as OCM [ManagedClusters](https://open-cluster-management.io/concepts/managedcluster/)
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+- [Helm](https://helm.sh/docs/intro/quickstart/#install-helm)
+
+## Configure OCM with RawFeedbackJsonString Feature Gate
+
+All OCM spoke clusters must be configured with the `RawFeedbackJsonString` feature gate set to true. This can be done in two ways:
+
+1. When running the `clusteradm join` command that joins the spoke cluster to the hub:
+
+```bash
+clusteradm join <snip> --feature-gates=RawFeedbackJsonString=true
+```
+
+2. By patching each spoke cluster's `klusterlet` in an existing OCM install:
+
+```bash
+kubectl patch klusterlet klusterlet --type merge --patch '{"spec": {"workConfiguration": {"featureGates": [{"feature": "RawFeedbackJsonString", "mode": "Enable"}]}}}' --context <EACH_SPOKE_CLUSTER>
+```
+
+## Installing MGC
+
+Run the following command in the context of your hub cluster to install the MGC [TODO: change build to Kuadrant/MGC repo/main]:
+
+```bash
+kustomize --load-restrictor LoadRestrictionsNone build "github.com/eoinfennessy/multicluster-gateway-controller.git/config/mgc-install-guide?ref=mgc_setup_guide_for_ocm" --enable-helm | kubectl apply -f -
+```
+
+In addition to the MGC, this will also install the Kuadrant add-on manager, Gateway API CRDs, and [cert-manager](https://cert-manager.io/).
+
+After the configuration has been applied, you can verify that the MGC and add-on manager have been installed and are running:
+
+```bash
+kubectl get pods -n multicluster-gateway-controller-system
+```
+```
+NAME                                           READY   STATUS    RESTARTS   AGE
+mgc-controller-manager-8659cfdf46-nwn4w        1/1     Running   0          2m
+mgc-kuadrant-add-on-manager-7479cb7f9c-rzk4w   1/1     Running   0          2m
+```
+
+## Creating a ManagedZone
+
+To manage the creation of DNS records, MGC uses [ManagedZone](https://docs.kuadrant.io/multicluster-gateway-controller/docs/how-to/managedZone/) resources. A `ManagedZone` can be configured to use DNS Zones on either AWS (Route53), and GCP. We will now create a ManagedZone on the cluster using AWS credentials.
+
+First, export the [environment variables detailed here](https://docs.kuadrant.io/multicluster-gateway-controller/docs/getting-started/#config) in a terminal session.
+
+Next, create a secret and configMap containing the AWS credentials and config [TODO: is the ConfigMap needed here?]:
```

Comment:
In my doc around providers, I will list the AWS and Google Perms we need so the user knows what to provide and what we can do